app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -97,6 +99,10 @@ fun MapScreen(
           DisposableMapEffect { mapView ->
             val viewAnnotationManager = mapView.viewAnnotationManager
 
+            // initialize the Gson object that will deserialize and serialize the parking to bind it
+            // to the marker
+            val gson = Gson()
```

Comment:
This could maybe be placed out of the scaffold? I'm not quite sure how many times it is called but it could help to place it as outside as possible

Reply:
Agree and fixed !